simulator/kruxsim/devices.py

```diff
@@ -36,48 +36,76 @@
     DOCK: (440, 820),
 }
 
-
-def with_prefix(device):
+##################################
+# Handle absolute path of assets #
+##################################
+dirname = os.path.dirname(os.path.abspath(__file__))
+
+def with_prefix(device: str):
+    """
+    Add prefix to device name. If its a maixpy one, add `maixpy_` string.
+    
+    :param device
+    :return str
+    """
     return device if device.startswith("maixpy_") else "maixpy_" + device
 
 
 images = {}
 
+def load_image(device: str):
+    """
+    Load an image asset as :mod:`pygame.Surface`
 
-def load_image(device):
+    :param device
+    :return pygame.Surface
+    """
     device = with_prefix(device)
     if device == PC:
         return None
     if device not in images:
-        images[device] = pg.image.load(
-            os.path.join("assets", "%s.png" % device)
-        ).convert_alpha()
+        asset_path = os.path.abspath(f"{dirname}/../assets/{device}.png")
+        images[device] = pg.image.load(asset_path).convert_alpha()
     return images[device]
 
 
 fonts = {}
 
 
-def load_font(device):
+def load_font(device: str):
+    """
+    Load a :mod:`pygame.freetype.Font`
+
+    :param device: str
+    :return pygame.freetype.Font
+    """
     device = with_prefix(device)
     if device not in fonts:
+        # Get the current dir of current file
+        # to dynamically get the absolute path of bdf font
```

Comment:
too much comments, the `dirname` already tells it will return the name of the directory of the file passed. `abspath` already tells it will return the absolute path of the file passed.